### PR TITLE
Test: Add bidder call type to debug info

### PR DESCRIPTION
**Pull request by marki1an**
Test for https://github.com/prebid/prebid-server-java/pull/1892

### CODE REVIEW

File: src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy [hunk 1]
```diff
@@ -304,4 +309,39 @@ class DebugSpec extends BaseSpec {
...
+
+    def "PBS should return HTTP call type when it's default bidder call"() {
+        given: "Default basic generic BidRequest"
+        def bidRequest = BidRequest.defaultBidRequest
+
+        when: "PBS processes auction request"
+        def response = defaultPbsService.sendAuctionRequest(bidRequest)
+
+        then: "Response should contain call type HTTP"
+        assert response.ext?.debug?.httpcalls[GENERIC.value].first().calltype == HTTP
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy:321]: It should be inverted. When it's a regular bidder call, `calltype` field should not be populated at all.

**Review by Net-burst — CHANGES_REQUESTED** [review]

File: src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy (hunk 1 above)
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy:321]: `calltype` should be in camelCase in the model.

Comment (marki1an) [reply src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy:321]: But in `ExtHttpCall.class` all fields are not in camelCase.

Comment (Net-burst) [reply src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy]: And that is the error, unfortunately. Model should adhere to Java code conventions.

File: src/main/java/org/prebid/server/proto/openrtb/ext/response/ExtHttpCall.java [hunk 2]
```diff
@@ -20,7 +20,7 @@ public class ExtHttpCall {
 
     String responsebody;
 
-    BidderCallType calltype;
+    BidderCallType callType;
```
Comment (Net-burst) [inline src/main/java/org/prebid/server/proto/openrtb/ext/response/ExtHttpCall.java:23]: Stop. You should never change production model for tests. And you should now use production model for tests too. Create a separate model for test.

**Review by Net-burst — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/proto/openrtb/ext/response/ExtHttpCall.java (hunk 2 above)
Comment (Net-burst) [reply src/main/java/org/prebid/server/proto/openrtb/ext/response/ExtHttpCall.java:23]: I did a typo there. I meant that you should not to touch production model in general.

Still, you will need to undo your changes to the prod code.

**Review by Net-burst — APPROVED** [review]
LGTM

**Pull request merged**